README.md

```diff
@@ -8,10 +8,10 @@ The `contxt-sdk` can be installed with NPM:
 npm install --save @ndustrial/contxt-sdk
 ```
 
-There are two peer dependencies for `contxt-sdk`, `auth0-js` and `axios`. If you don't already have a compatible version installed, run:
+There are three peer dependencies for `contxt-sdk`, `auth0-js`, `axios`, and `ws`. If you don't already have a compatible version installed, run:
 
 ```bash
-npm install --save auth0-js@^9.0.0 axios@~0.17.0
+npm install --save auth0-js@^9.0.0 axios@~0.17.0 ws
```

Comment:
Should we make this `ws@~6.1.3`? If `ws` is following semver this will allow patch versions but lock it into the minor version.

Reply:
Sure! That's fine with me!